src/main/resources/apiKey.json

```diff
@@ -0,0 +1,3 @@
+{
+    "apiKey" :"AIzaSyBrrPrvZARKr5QJVV1W17ngKB5YdMTR_nw"
```

Comment:
Make sure that there are adequate restrictions on this API key in the cloud console. Now that this is public on github, it is at risk of anyone using it (and charging to your project).

Restrictions will limit the API key so that it can only be used from known domains etc, and will be enforced by the Maps API server.

Reply:
I will add the restrictions, but is there a way to hide the API key file? It has to be stored in GitHub for the tests to pass and from what I found online, using  .gitignore will not work.